From 1a6c73071593f804de9a504529639c9b5768e790 Mon Sep 17 00:00:00 2001
From: Jonathan Bell <jonathan@raspberrypi.com>
Date: Wed, 24 Jan 2024 16:26:37 +0000
Subject: [PATCH] dt-bindings: usb: update dwc3 bindings for parkmode-disable
 quirks

There are three disable bits, one for each bus-instance type. Add a
quirk to cover the FS/LS type, and update the slightly mangled quirk
descriptions in the process.

Signed-off-by: Jonathan Bell <jonathan@raspberrypi.com>
---
 Documentation/devicetree/bindings/usb/snps,dwc3.yaml | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

--- a/Documentation/devicetree/bindings/usb/snps,dwc3.yaml
+++ b/Documentation/devicetree/bindings/usb/snps,dwc3.yaml
@@ -234,12 +234,17 @@ properties:
 
   snps,parkmode-disable-ss-quirk:
     description:
-      When set, all SuperSpeed bus instances in park mode are disabled.
+      When set, disable park mode for all Superspeed bus instances.
     type: boolean
 
   snps,parkmode-disable-hs-quirk:
     description:
-      When set, all HighSpeed bus instances in park mode are disabled.
+      When set, disable park mode for all Highspeed bus instances.
+    type: boolean
+
+  snps,parkmode-disable-fsls-quirk:
+    description:
+      When set, disable park mode for all Full/Lowspeed bus instances.
     type: boolean
 
   snps,dis_metastability_quirk:
